src/components/Reports/GoalCalculator/SummaryReport/Steps/PresentingYourGoalStep/PresentingYourGoal.test.tsx

```diff
@@ -0,0 +1,128 @@
+import { GetUserQuery } from 'src/components/User/GetUser.generated';
+import theme from 'src/theme';
+import { GoalCalculatorProvider } from '../../../Shared/GoalCalculatorContext';
+import { PresentingYourGoal } from './PresentingYourGoal';
+
+/*
+ * Mocking recharts ResponsiveContainer to avoid ResponsiveContainer
+ * width and height issue
+ * https://jskim1991.medium.com/react-writing-tests-with-graphs-9b7f2c9eeefc
+ */
+jest.mock('recharts', () => {
+  const OriginalModule = jest.requireActual('recharts');
+  return {
+    ...OriginalModule,
+    ResponsiveContainer: ({ children }) => (
+      <OriginalModule.ResponsiveContainer width={800} height={800}>
+        {children}
+      </OriginalModule.ResponsiveContainer>
+    ),
+  };
+});
+
+jest.mock('src/hooks/useOrganizationId', () => ({
+  useOrganizationId: jest.fn(() => 'organization-id-1'),
+}));
+
+const TestComponent: React.FC = () => (
+  <SnackbarProvider>
+    <ThemeProvider theme={theme}>
+      <GqlMockedProvider<{
+        GetUser: GetUserQuery;
+        GetUsersOrganizationsAccounts: GetUsersOrganizationsAccountsQuery;
+      }>
+        mocks={{
+          GetUser: {
+            user: {
+              id: 'account-list-id-1',
+              firstName: 'Obiwan',
+              lastName: 'Kenobi',
+            },
+          },
+          GetUsersOrganizationsAccounts: {
+            userOrganizationAccounts: [
+              {
+                id: 'user-org-account-id-1',
+                username: 'obiwan.kenobi',
+                latestDonationDate: '2023-12-01',
+                lastDownloadedAt: '2023-12-15',
+                organization: {
+                  id: 'organization-id-1',
+                  name: 'Campus Crusade for Christ, Inc.',
+                  apiClass: 'DataSync',
+                  oauth: false,
+                },
+              },
+            ],
+          },
+        }}
+      >
+        <GoalCalculatorProvider>
+          <PresentingYourGoal />
+        </GoalCalculatorProvider>
+      </GqlMockedProvider>
+    </ThemeProvider>
+  </SnackbarProvider>
+);
+
+describe('PresentingYourGoal', () => {
+  beforeEach(() => {
+    beforeTestResizeObserver();
+  });
+
+  it('renders cell text', () => {
+    const { getByRole } = render(<TestComponent />);
+    expect(
+      getByRole('heading', { name: 'Personal Information' }),
+    ).toBeInTheDocument();
+    expect(getByRole('cell', { name: 'User' })).toBeInTheDocument();
+    expect(getByRole('cell', { name: 'Mission Agency' })).toBeInTheDocument();
+    expect(
+      getByRole('heading', { name: 'Monthly Support Needs' }),
+    ).toBeInTheDocument();
+
+    expect(
+      getByRole('heading', { name: 'Monthly Support Breakdown' }),
+    ).toBeInTheDocument();
+  });
+
+  it("renders the user's name", async () => {
+    const { getAllByTestId } = render(<TestComponent />);
+    await waitFor(() => {
```

Comment:
Would it work to look for something with the role `cell` and the name `Obiwan Kenobi`?